receiver/jaegerreceiver/trace_receiver_test.go

```diff
@@ -18,9 +18,13 @@ import (
 	"context"
 	"encoding/binary"
 	"fmt"
+	"path"
 	"testing"
 	"time"
 
+	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
```

Comment:
Shall the imports be organized following this convention: 1st: standard lib; 2nd: third-parties; 3rd: otel-svc? So the above line could be moved to the end.

Reply:
Yes - these imports were auto-imported. Will fix them up.